climakitae/data_export.py

```diff
@@ -163,32 +142,40 @@ def _export_to_user(user_export_format, data_to_export,
     data_to_export: xarray ds or da to export
     file_name: string corresponding to desired output file name
     kwargs: variable, scenario, and simulation (as needed)
-    """
+    """   
+    ftype = str(type(data_to_export)).strip('<class>')
+    
+    if "xarray" not in ftype:
+            raise Exception("Cannot export object of type " + ftype
+                            + ". Please pass an xarray dataset"
+                            + " or data array.")
+            
     ndims = len(data_to_export.dims)
     file_name = file_name.split('.')[0]
 
-    assert type(file_name) is str,("Please pass a string"
-        + " (any characters surrounded by quotation marks)"
-        + " for your file name.")
+    if type(file_name) is not str:
+        raise Exception("Please pass a string"
+                        + " (any characters surrounded by quotation marks)"
+                        + " for your file name.")
 
     path_and_name = './' + file_name
     req_format = user_export_format.output_file_format
     if req_format is None:
-        raise AssertionError("Please select a file format"
-                             + " from the dropdown menu.")
+        raise Exception("Please select a file format"
+                        + " from the dropdown menu.")
 
     extension_dict = {'NetCDF' : '.nc',
                       'CSV' : '.csv.gz',
                       'GeoTIFF' : '.tif'}
 
     f_extension = extension_dict[req_format]
```

Comment:
f_extension isn't used more than once so you can just move the `extension_dict` to the line below...